### PR TITLE
[DESKTOP] Add dark theme

### PR DESCRIPTION
**Pull request by jvnm-dev**
This commit add a dark theme.
We need to define a way to switch to dark theme.

### CODE REVIEW

**Comment by jvnm-dev** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>API changes but no tests!</td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>You should add an entry into CHANGELOG.md!</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>Hi there, thanks for contributing!</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against cade3158d359a7dd8a038cc9ca43756699192079
</p>

**Comment by jvnm-dev** [comment]
image

image

File: projects/desktop/src/helpers/logo.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import logoDark from '../assets/img/logo-small-dark.png'
+import { ITheme } from './../interfaces/theme'
+
+export const getLogo = (theme: ITheme): string => {
+    switch (theme.mode) {
+    case 'classic':
+        return logoClassic
+    case 'dark':
+        return logoDark
+    default:
```
Comment (Sylv11) [inline projects/desktop/src/helpers/logo.ts:11]: Indentation

Comment (jvnm-dev) [reply projects/desktop/src/helpers/logo.ts:11]: eslint

Comment (Sylv11) [reply projects/desktop/src/helpers/logo.ts:11]: Talk better

**Review by Sylv11 — APPROVED** [review]

**Pull request merged**